jwst/flatfield/flat_field.py

```diff
@@ -1523,10 +1585,13 @@ def interpolate_flat(image_flat, image_dq, image_wl, wl):
 
     flat_dq : ndarray, 2-D, uint32
         The data quality array corresponding to `flat_2d`.
+
+    flat_err : ndarray, 2-D, float32
+        The error array corresponding to `flat_2d`.
     """
 
     if len(image_flat.shape) < 3:
-        return (image_flat, image_dq)
+        return image_flat, image_dq, image_err
```

Comment:
Lines 1599 and 1601 & 1602 need to return three items.

Reply:
Fixed